src/clipboard-action.js

```diff
@@ -41,6 +41,8 @@ class ClipboardAction {
         this.fakeElem = document.createElement('input');
         this.fakeElem.style.position = 'absolute';
         this.fakeElem.style.left = '-9999px';
+        this.fakeElem.style.width = 0;
+        this.fakeElem.style.height = 0;
```

Comment:
This does not work. Setting to 0px / 0em doesn't work as this gives a negative w/h on some browsers http://stackoverflow.com/a/30810322/981073